src/components/project/dto/create-project.dto.ts

```diff
@@ -31,6 +37,9 @@ export abstract class CreateProject {
 
   @Field(() => ProjectStep, { nullable: true })
   readonly step?: ProjectStep;
+
+  @Field(() => Sensitivity, { nullable: true })
```

Comment:
Can you add description here and update that says this is only for internship projects

Reply:
Added to both